src/crypto/chacha20.h

```diff
@@ -20,7 +20,9 @@ class ChaCha20
     void SetKey(const unsigned char* key, size_t keylen);
     void SetIV(uint64_t iv);
     void Seek(uint64_t pos);
-    void Output(unsigned char* output, size_t bytes);
+
+    void Output(unsigned char* c, size_t bytes);
```

Comment:
nit: This function should be renamed `Keystream` now that the class is not just used for PRNG

Reply:
Renamed to `Keystream`